src/imars3d/backend/diagnostics/tilt.py

```diff
@@ -45,6 +47,10 @@ def find_180_deg_pairs_idx(
         raise ValueError("angles must be a 1d array")
     # ensure angles are in degrees
     angles = angles if in_degrees else np.degrees(angles)
+    # compute atol if not specified
+    if atol is None:
+        atol = np.min(np.absolute(np.diff(angles))) / 2.0
```

Comment:
Since `angles` is relatively small, couldn't the code either check if it is sorted or just out-of-place indexing. Something like
```python
sorted_indices = np.argsort(angles)
atol = np.min(np.diff(angles[sorted_indices])) / 2.
del sorted_indices
```
This shouldn't ever need to have the `np.absolute` since the array will be in increasing order. It would be better to throw a `RuntimeError` or `ValueError` in that case anyhow.